extensions/airsy/src/components/IndexDetails.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from "react";
+import { List } from "@raycast/api";
+import translateAndColor from "../helpers/translateAndColor";
+
+const IndexDetails: FC<{ title: string; indexValue: string | undefined; value?: number }> = ({
+  title,
+  indexValue,
+  value,
+}) => {
+  const text = translateAndColor(indexValue).text;
+  const color = translateAndColor(indexValue).color;
+
+  return (
+    <>
+      <List.Item.Detail.Metadata.Label title={title} />
+      <List.Item.Detail.Metadata.TagList title="Index">
+        <List.Item.Detail.Metadata.TagList.Item text={text} color={color} />
+      </List.Item.Detail.Metadata.TagList>
+      {value && (
```

Comment:
```suggestion
      {typeof value === "number" && (
```
Should fix the error: A `Metadata` only accepts `Metadata.Label`, `Metadata.Link`, `Metadata.Separator` and `Metadata.TagList` children. Ignoring "string".